src/lib/minions/functions/removeFoodFromUser.ts

```diff
@@ -38,6 +38,13 @@ export default async function removeFoodFromUser({
 		totalHealingNeeded = reduceNumByPercent(totalHealingNeeded, 17.5);
 		reductions.push(`-17.5% for Ely ${Emoji.Ely}`);
 	}
+	if (
+		gearSetupsUsed.some(i => i[0] === 'melee') &&
+		rawGear.melee.hasEquipped(['Justiciar faceguard', 'Justiciar chestguard', 'Justiciar legguards'], true, true)
+	) {
+		totalHealingNeeded = reduceNumByPercent(totalHealingNeeded, 12.5);
+		reductions.push('-6% for Justiciar');
+	}
```

Comment:
If +k costs Sang charges, then maybe give a food reduction for using Sang, since it heals on 1/6 of attacks, 50% of the damage. 

Max hit is 43 with imbued heart, so lets say 40:
```ts
const maxHit = 40;
const heals = Math.floor(sangChargesUsed / 6));
const healAmount = heals * (maxHit / 2);
totalHealingNeeded -= healAmount;
```